### DISCUSSION TITLE
Getting no changes detected

### DISCUSSION THREAD

**Discussion by ghost** [stefanzweifel/git-auto-commit-action#150] [Help]
Hi! I am on a private company repository and got the following messages:

`INPUT_REPOSITORY value: .`
`fatal: Not a git repository (or any parent up to mount point /__w)`

I did put a **PAT** at `token`, but still gets this.

What's wrong with?

**Comment by stefanzweifel** [comment]
Are you doing something special in the `actions/checkout` step? Like do you move the checked out repository to a different place?

`INPUT_REPOSITORY: .` basically means that `git-auto-commit` is looking for a git repository in the root location.

**Reply by ghost** [reply]
Not really. I just do some lint changes and run tests before it then it always shows this fatal error on its step.

`- name: COMMIT`
     ` if: success()`
     ` uses: stefanzweifel/git-auto-commit-action@v4`
     ` with:`
        `commit_message: Apply linting changes`

**Reply by stefanzweifel** [reply]
Hm, it's really hard to help here with that limited information.
Mabye the current working directory `pwd` is not correct when `git-auto-commit-action` is invoked? 🤔 

Searching for this error also doesn't really yield any good solutions.
https://www.google.com/search?client=firefox-b-d&q=%22fatal%3A+Not+a+git+repository+%28or+any+parent+up+to+mount+point%22
